### ISSUE TITLE
Define a List (array) of Map-objects - not a Map - for the Verification: sendCodeAttempts

### ISSUE THREAD

**Issue by tuddman** [twilio/twilio-java#517]
### Issue Summary
I'm getting a deserialization exception when trying to parse the _array_ of `send_code_attempts`

where the  object mapper encounters an array 

https://github.com/twilio/twilio-java/blob/master/src/main/java/com/twilio/rest/verify/v2/service/Verification.java#L326

but is expecting a single Map:

https://github.com/twilio/twilio-java/blob/master/src/main/java/com/twilio/rest/verify/v2/service/Verification.java#L210

please correct to be `List<Map<String, Object>>`

### Exception/Log
```
Execution error (MismatchedInputException) at com.fasterxml.jackson.databind.exc.MismatchedInputException/from (MismatchedInputException.java:59).
Cannot deserialize instance of `java.util.LinkedHashMap<java.lang.Object,java.lang.Object>` out of START_ARRAY token
 at [Source: (ByteArrayInputStream); line: 1, column: 100] (through reference chain: com.twilio.rest.verify.v2.service.Verification["send_code_attempts"])
```

### Technical details:
* twilio-java version:   7.47.4

**Comment by childish-sambino** [comment]
I'm seeing an array of objects coming back from the API so looks like this is a bug. Since this is generated code the upstream definition will need to be updated.

**Comment by tuddman** [comment]
`7.47.3`  works ... without sendCodeAttempts.   looks like it was introduced with [this commit](https://github.com/twilio/twilio-java/commit/7a4bb77385aab53ee2d000c17aefe0175cf36029)

out of interest, how are you generating the code?  where is the actual upstream definition?

**Comment by childish-sambino** [comment]
There's internal tooling and an API spec which is used to generated all the files with a `Yoyodyne` header.

There's a change to fix this issue which is currently under review. Hope to get it included in the release today, if not then the following release after that. As you mentioned, pinning to version `7.47.3` can be used as a temporary workaround.

**Comment by philnash** [comment]
Just noting, [this has also been reported on Stack Overflow here](https://stackoverflow.com/questions/60264085/phone-verification-with-the-twilio-api-using-java?noredirect=1#comment106667093_60264085). I'll follow this issue and update SO when it is fixed.

**Comment by childish-sambino** [comment]
Merged upstream. Will be included in next release scheduled for 2020-03-04.

**Comment by eshanholtz** [comment]
Fixed in version 7.47.6 released on 2020-03-04.
